moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -97,127 +102,60 @@ const handleColumnChange = ({ field }, hidden) => {
 };
 
-const useFilterQuery = (locationSearch) =>
-  useMemo(() => {
-    return new URLSearchParams(locationSearch);
-  }, [locationSearch]);
-
 /**
- * if filter exists in url, decodes base64 string and returns as object
- * Used to initialize filter state
- * @return Object
+ * Returns a ProjectStatusBadge component based on the status and phase of project
+ * @param {string} phase - A project's current phase
+ * @param {number} statusId - Project's status id
+ * @return {JSX.Element}
  */
-const useMakeFilterState = (filterQuery) =>
-  useMemo(() => {
-    if (Array.from(filterQuery).length > 0) {
-      try {
-        return JSON.parse(atob(filterQuery.get("filter")));
-      } catch {
-        return {};
-      }
-    }
-    return {};
-  }, [filterQuery]);
+const buildStatusBadge = ({ phaseName, phaseKey }) => (
+  <ProjectStatusBadge phaseName={phaseName} phaseKey={phaseKey} condensed />
+);
 
 /**
  * GridTable Search Capability plus Material Table
  * @param {Object} query - The GraphQL query configuration
- * @param {String} searchTerm - The initial term
  * @return {JSX.Element}
  * @constructor
  */
 const ProjectsListViewTable = ({ query }) => {
   const classes = useStyles();
 
-  // create URLSearchParams from url
-  const navSearchTerm = useLocation()?.state?.searchTerm;
-  const filterQuery = useFilterQuery(useLocation().search);
-  const initialFilterState = useMakeFilterState(filterQuery);
-
-  /**
-   * Stores the string to search for and the column to search against
-   * @type {Object} search
-   * @property {string} value - The string to be searched for
-   * @property {string} column - The name of the column to search against
-   * @function setSearch - Sets the state of search
-   * @default {{value: "", column: ""}}
-   */
-  const [search, setSearch] = useState({
-    value: navSearchTerm ?? "",
-    column: "",
-  });
-
   // anchor element for advanced search popper in Search to "attach" to
   // State is handled here so we can listen for changes in a useeffect in this component
   const [advancedSearchAnchor, setAdvancedSearchAnchor] = useState(null);
 
-  /**
-   * Stores objects storing a random id, column, operator, and value.
-   * @type {Object} filters
-   * @function setFilter - Sets the state of filters
-   * @default {if filter in url, use those params, otherwise {}}
-   */
-  const [filters, setFilter] = useState(initialFilterState);
-
   const [hiddenColumns, setHiddenColumns] = useState(
     JSON.parse(localStorage.getItem("mopedColumnConfig")) ?? DEFAULT_HIDDEN_COLS
   );
 
-  /**
-   * Query Management
-   */
-
-  // Resets the value of "where" "and" "or" to empty
-  query.cleanWhere();
-
-  // For each filter added to state, add a where clause in GraphQL
-  // Advanced Search
-  Object.keys(filters).forEach((filter) => {
-    let { envelope, field, gqlOperator, value, type, specialNullValue } =
-      filters[filter];
+  /* Project list query */
```

Comment:
This is where I decided to move the hooks that build the separate pieces of the query outside of `useProjectListView`. See https://github.com/cityofaustin/atd-moped/pull/1144#discussion_r1343210020.

Reply:
good stuff!